Makefile

```diff
@@ -0,0 +1,28 @@
+local="http://localhost:8080/ipfs/"
+gway="https://ipfs.io/ipfs/"
+domain="refs.ipfs.io"
+record="dmca.denylists.lists"
```

Comment:
o/ that's interesting. i would've expected

```
domain="denylists.lists.refs.ipfs.io"
record="dmca"
```
does dns have a limit i'm not aware of?